src/containers/run.c

```diff
@@ -189,11 +189,8 @@ void run_container_offset(const run_container_t *c, container_t **loc,
 
 /* Free memory. */
 void run_container_free(run_container_t *run) {
-    if (run->runs !=
-        NULL) {  // Jon Strabala reports that some tools complain otherwise
-        roaring_free(run->runs);
-        run->runs = NULL;  // pedantic
-    }
+    roaring_free(run->runs);
+    run->runs = NULL;  // pedantic
```

Comment:
I think especially in these cases where the next line (in a 3 line function) clearly means the value set here doesn't matter, we shouldn't bother setting it to null.

Same above in array_container_free

Reply:
I thought I had removed the pedantic settings. Let me solve this.

Reply:
I pushed. Resolved.